modules/ocf_mirrors/manifests/centos.pp

```diff
@@ -8,10 +8,19 @@
     recurse => true,
   }
 
-  ocf_mirrors::monitoring { 'centos':
+  ocf_mirrors::monitoring {
+    default:
       type          => 'unix_timestamp',
       upstream_host => 'mirror.centos.org',
-      ts_path       => 'TIME',
+      ts_path       => 'TIME';
+
+    'centos':
+    ;
+
+    'centos-altarch':
+      filename      => 'health-altarch',
+      upstream_path => '/altarch',
+      project_path  => '/opt/mirrors/project/centos';
```

Comment:
Why not just put this in its own project? It has its own sync script, monitoring, directory on mirrors.o.b.e. Seems to me like it should just be separated out.

Reply:
Cool. That'll actuallt make the puppet code a lot simpler too.

Reply:
Sounds good to me, sure

Reply:
If you're gonna do that, you might as well consider throwing in ubuntu-releases and trisquel-images into the mix.

Reply:
@kpengboy Sure. Looks like (this)[https://mirrors.ocf.berkeley.edu/trisquel-images/project/trace/mirrors.ocf.berkeley.edu] should work for trisquel. Can't seem to find any file to healthcheck in ubuntu releases. If you find one I'd be happy to add it.